modules/ebi-metagenomics/cat/meta.yml

```diff
@@ -0,0 +1,65 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "cat"
+description: MGnify CAT based bins taxonomy assignation tool
+keywords:
+  - mags
+  - taxonomy
+  - genomics
+tools:
+  - "cat":
+      description: "CAT/BAT: tool for taxonomic classification of contigs and metagenome-assembled genomes (MAGs)"
+      homepage: "https://github.com/MGXlab/CAT_pack"
+      documentation: "https://github.com/MGXlab/CAT_pack"
+      tool_dev_url: "https://github.com/MGXlab/CAT_pack"
+      doi: "10.1038/s41467-024-47155-1."
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+
+  - contigs_fasta:
+      type: file
+      description: The fasta file that contains contigs
+      pattern: "*.{fa,fasta}"
+
+  - cat_db:
+      type: directory
+      description: Path to directory that contains CAT database files
+
+  - tax_db:
```

Comment:
rename to taxonomy_db (as in channel above, or the other way around)

Reply:
done